src/read/elf/note.rs

```diff
@@ -143,17 +143,19 @@ impl<'data, Elf: FileHeader> Note<'data, Elf> {
         &self,
         endian: Elf::Endian,
     ) -> Option<GnuPropertyIterator<'data, Elf::Endian>> {
-        if self.name() != elf::ELF_NOTE_GNU || self.n_type(endian) != elf::NT_GNU_PROPERTY_TYPE_0 {
-            return None;
+        match (self.n_type(endian), self.name()) {
```

Comment:
Please leave it as the if statement because it nests less.

Reply:
Done.

Reply:
Please remove this change completely.

Reply:
Done.